flowey/flowey_lib_hvlite/src/download_openhcl_kernel_package.rs

```diff
@@ -99,7 +99,15 @@ impl FlowNode for Node {
                 match arch {
                     OpenhclKernelPackageArch::X86_64 => "x64",
                     OpenhclKernelPackageArch::Aarch64 => "arm64",
-                }
+                },
+                match kind {
```

Comment:
This is temporary. Releases have moved from `.zip` to `.tar.gz`. As soon as a new dev kernel is released, we can switch both over to using tar.gz.